index.html

```diff
@@ -116,7 +116,33 @@ <h2 class="about-title">About me</h2>
 <p class="about-contend">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 <a href="#">Get My Resume</a>
 <div class="ornament2" ></div>
-
 </section>
+
+
+
+
+<footer class="form">
```

Comment:
- [ optional] The contact section is a section on its own. the form can be inside not the footer.